.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build images
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        suffix: ['', '-headless']
+    steps:
+      - uses: actions/checkout@v2
+      - id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: oddbirds/pyjs
+          tags: |
+            type=ref,event=branch,suffix=${{ matrix.suffix }}
+            type=ref,event=tag,suffix=${{ matrix.suffix }}
+            type=ref,event=pr,suffix=${{ matrix.suffix }}
+      - uses: docker/setup-qemu-action@v1
+      - id: buildx
+        uses: docker/setup-buildx-action@v1
+      - uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          file: Dockerfile${{ matrix.suffix }}
+          platforms: linux/amd64,linux/arm64
```

Comment:
I assume `linux/arm64` works without specifying `linux/arm64/v8`? (That's what I see on e.g. https://hub.docker.com/_/python?tab=tags)

Reply:
That's an interesting question. I found this in the [Known Issues](https://docs.docker.com/desktop/mac/apple-silicon/#known-issues) section of the docs for Apple Silicon:

> We recommend running arm64 containers on Apple Silicon machines whenever possible, and encouraging container authors to produce arm64, or multi-arch, versions of their containers.

So it seems like `arm64` is what we want for M1 Macs. In [this answer](https://stackoverflow.com/a/64635585/1330003) `arm64/v8` is listed as an alias of `arm64`, so I think Docker knows what we mean.